setup.py

```diff
@@ -11,14 +11,13 @@ def read(fname):
     name='calfire-wildfires',
     version='0.0.4',
     description="Download wildfires data from CalFire",
-    long_description=read('README.rst'),
+    long_description=read('README.md'),
     author='Los Angeles Times Data Desk',
```

Comment:
One very nerdy trick is for Markdown to work with PyPI you have to set the long_description_content_type as see [here](https://github.com/palewire/nws-wwa/blob/main/setup.py#L15)

Reply:
Added!